index.html

```diff
@@ -4,6 +4,8 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Favicons -->
+    <link rel="icon" type="image/png" href="./assets/favicon.ico" />
```

Comment:
I think you can move this favicon link to your main css, that way you wouldn't have to link it in all of your html

Reply:
The favicon can be linked only in the header.